java/code/src/com/redhat/rhn/domain/org/test/OrgFactoryTest.java

```diff
@@ -188,11 +188,9 @@ public void testOrgDefautRegistrationToken() throws Exception {
         orig = OrgFactory.save(orig);
         Long origId = orig.getId();
         flushAndEvict(orig);
-
         Org lookup = OrgFactory.lookupById(origId);
         assertEquals(token.getId(), lookup.getToken().getId());
         lookup.setToken(null);
-        flushAndEvict(lookup);
```

Comment:
have you tried to bring this also back? Maybe it works as well

Reply:
not this one: 
```
javax.persistence.EntityExistsException: A different object with the same identifier value was already associated with the session : [com.redhat.rhn.domain.org.Org#406]
```